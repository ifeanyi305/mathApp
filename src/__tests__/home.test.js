import { render } from '@testing-library/react';
import HomePage from '../components/home';

describe('test Homepage component', () => {
  it('renders correctly', () => {
    const tree = render(<HomePage />);
    expect(tree).toMatchSnapshot();
  });
});
