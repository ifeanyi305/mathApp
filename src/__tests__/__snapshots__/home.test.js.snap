// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Homepage component renders correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="home"
      >
        <h1>
          Welcome to our page
        </h1>
        <div
          class="content"
        >
          <p>
            Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi dicta earum, omnis odio sed pariatur sapiente consequuntur natus consectetur maxime error, aliquam, laborum repellat magni reiciendis asperiores officia. Animi, impedit?
          </p>
          <p>
            Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et ullam eius, perferendis exercitationem praesentium fuga saepe dolores voluptatem reiciendis nam quam perspiciatis ad, repudiandae molestiae esse voluptatibus consequuntur, eligendi molestias?
          </p>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="home"
    >
      <h1>
        Welcome to our page
      </h1>
      <div
        class="content"
      >
        <p>
          Lorem ipsum dolor sit amet consectetur adipisicing elit. Excepturi dicta earum, omnis odio sed pariatur sapiente consequuntur natus consectetur maxime error, aliquam, laborum repellat magni reiciendis asperiores officia. Animi, impedit?
        </p>
        <p>
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Et ullam eius, perferendis exercitationem praesentium fuga saepe dolores voluptatem reiciendis nam quam perspiciatis ad, repudiandae molestiae esse voluptatibus consequuntur, eligendi molestias?
        </p>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
